libs/javascript.json

```diff
@@ -71,16 +71,21 @@
   {"id":"js.vuex","imports":["vuex"],"name":"vuex","repo":"vuejs/vuex","tags":["state-management"],"tech":["javascript-web"],"status":"model-ready"},
   {"id":"js.ml5","imports":["ml5"],"name":"ml5","repo":"ml5js/ml5-library","tags":["toolkit"],"tech":["ml"],"status":"awaiting-model"},
   {"id":"js.webpack","imports":[],"name":"webpack","repo":"webpack/webpack","tags":["module-bundler"],"tech":["software-distribution"],"status":"awaiting-model"},
-  {"id":"js.mariadb","imports":["mariadb"],"name":"mariadb","repo":"MariaDB/mariadb-connector-nodejs","tags":["sql"],"tech":["storage"],"status":"awaiting-model"},
+  {"id":"js.knex","imports":["knex"],"name":"knex","repo":"tgriesser/knex","tags":["db-abstraction-layer","db-manipulation"],"tech":["storage"],"status":"awaiting-model"},
```

Comment:
we really use only the first tag. it does not hurt to have multiple but it's just that they are not in use at the moment.